data/avro-schema/src/main/resources/avro/net/corda/data/membership/db/request/command/AddPreAuthToken.avsc

```diff
@@ -0,0 +1,27 @@
+{
+  "type": "record",
+  "name": "AddPreAuthToken",
+  "namespace": "net.corda.data.membership.db.request.command",
+  "doc": "Add a pre auth token",
+  "fields": [
+    {
+      "name": "tokenId",
+      "doc": "Token identifier of the pre auth token.",
+      "type": "string"
+    },
+    {
+      "name": "ownerX500Name",
+      "doc": "X500 name of the owner of the pre auth token.",
+      "type": "string"
+    },
+    {
+      "name": "ttl",
```

Comment:
Missing doc. It would be useful to indicate here if the TTL is milliseconds/seconds/minutes